docker-compose.yml

```diff
@@ -67,7 +67,9 @@ services:
 
   ngrok:
     image: wernight/ngrok
-    command: ngrok http app:8000
+    env_file:
+      - env.d/ngrok
+    command: sh -c "if [[ $$NGROK_AUTH_TOKEN ]]; then ngrok authtoken $$NGROK_AUTH_TOKEN; fi && ngrok http app:8000"
```

Comment:
```suggestion
    command: ngrok http app:8000
```

For what I understand in the image documentation, there is no need to do that. In the entrypoint, it checks if the variable `NGROK_AUTH` exists and use it if it exists. So IMO there is nothing more to do than declaring the environment variable : https://github.com/wernight/docker-ngrok/blob/7fc977ff0458adc955dec7ec38d1906a22e0431c/entrypoint.sh#L37-L40